### PR TITLE
openrgb: 0.7 -> 0.8

### PR DESCRIPTION
**Pull request by SuperSamus**
Diff: https://gitlab.com/CalcProgrammer1/OpenRGB/-/compare/release_0.7...release_0.8

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/openrgb/default.nix [hunk 1]
```diff
@@ -1,35 +1,23 @@
...
+  buildInputs = [ libusb1 hidapi mbedtls_2 qtbase qttools ];
 
-  installPhase = ''
-    runHook preInstall
-
-    mkdir -p $out/bin
-    cp openrgb $out/bin
-
-    substituteInPlace 60-openrgb.rules \
+  preInstall = ''
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/openrgb/default.nix:17]:
```suggestion
  postPatch = ''
```
or do these files not exist at this point?

**Comment by deejayem** [comment]
Result of `nixpkgs-review pr 204820` run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>openrgb</li>
  </ul>
</details>

**Comment by RicArch97** [comment]
Got this information from an OpenRGB developer when i was trying to package this update myself;
The `60-openrgb.rules` should be included in the build, by running a script. This script should however also contain a commit ID, which can be used for trouble shooting purposes. Can you confirm that the commit ID exists in this file?

> the script is called with the commit ID and source path.
>
>Taken from OpenRGB.pro
>udev_rules.extra    = $$PWD/scripts/build-udev-rules.sh $$PWD $$GIT_COMMIT_ID
>
> The source path is no big deal if you're building in place . The Commit ID stamps the outfile rules with the commit ID
> to know which build it belongs to for troubleshooting purposes

Might not be a big deal but mentioning it just in case. Haven't had time to tested the PR yet.

**Review by deejayem — APPROVED** [review]
Built on aarch64-linux, and tested that the binary can print its help message. On x64_64-linux I was able to launch the GUI.

**Comment by SuperSamus** [comment]
> Can you confirm that the commit ID exists in this file?

I checked, and there is no commit.
I could fix this with `leaveDotGit = true;` and by adding `git` to `nativeBuildInputs`. Or is there a better solution? (Replacing the git commands in `OpenRGB.pro` would be harder to maintain.)

**Comment by SuperSandro2000** [comment]
leaveDotGit has some issues and should generally be avoided if possible.

**Pull request merged**